### DISCUSSION TITLE
Copilot shuts itself down Issue

### DISCUSSION THREAD

**Discussion by ChichoRD** [community/community#31798] [Copilot Conversations]
Hello, after trying the "beta" version of copilot for some months and having my free trial ended afterwards when it released officialy, I afortunately accounted on my current bachelorette degree to gain student access to it as its features truly  amazed me when it was in pre-release.

However, **soon after I activated my student license** and got copilot this way (this could be a coincidence), it did not behave the same as it has done in the past in my IDE (Visual Studio 2022, latest version). **The issue is that right now I can open any solution, copilot will start working and suggesting code but then it will stop doing any of that after about 10 suggestions. If I restart my IDE I'll get another bunch of suggestions but only for a moment before it apperently stops working.** The little copilot head would stop spinning and not "think" again no matter if I trigger the suggestion command or if I start coding again, unless as I said I restart the editor.

Also it may be worth mentioning that I have not been able to see any suggestion in the copilot window lately even before the issue occurs (I relied more on the in-line suggestions than in the window but I guess it is not working neither). Furthermore, unlike other issues mentioned in other discussions, **my github copilot output window does not show any error message**.
To fix it **I have tried uninstalling the extension, updating my IDE to latest version** and prompting it to suggest me anything.

I really like the extension and hope to be using it again soon, but restarting my editor every time I want a suggestion is not a viable solution for me. Should you need any more information to track my problem, I will provide it as needed.

Thanks.

**Comment by caseylin2002** [comment]
I also encountered the same problem, not only in Visual Studio Code, but also in JetBrains software, and I tried the old version of the plug-in and did not have this problem. I also contacted the community, but the relevant answer was that I was asked to upgrade the new version, and the new version had a very serious BUG, that is, it closed itself inexplicably.
